releso/util/visualization.py

```diff
@@ -258,23 +301,197 @@ def plot_episode_log(
     fig.update_yaxes(title_text="Seconds<br>per step [s]", row=5, col=1)
     fig.update_yaxes(title_text="Wall time [h]", row=6, col=1)
 
-    # Export the plot as the suffix or as "episode_log.html"
-    suffix = export_path.suffix
-    if suffix == ".html":
-        fig.write_html(export_path)
-    elif suffix in [".png", ".jpg", ".jpeg", ".webp", ".svg", ".pdf"]:
-        fig.write_image(export_path)
-    elif export_path.is_dir():
-        fig.write_html(export_path / "episode_log.html")
-    else:
-        if not (export_path.parent / "episode_log.html").exists():
-            print(
-                "File path suffix is not known nor is the file path a directory. "
-                f"Saving to {export_path.parent / 'episode_log.html'}"
-            )
-            fig.write_html(export_path.parent / "episode_log.html")
-        else:
-            print(
-                "File path suffix is not know nor is the file path a "
-                "directory. Not saving."
+    return fig
+
+
+def plot_step_log(
+        step_log_file: pathlib.Path,
+        env_id: int,
```

Comment:
```suggestion
        env_id: int = 0,
```

Would this make a big difference?